Code/main.cpp

```diff
@@ -115,7 +115,6 @@ int main(int argc, char *argv[]) {
     }
     CALI_MARK_END("comp");
 
-    CALI_MARK_BEGIN("correctness_check");
```

Comment:
This is already marked at https://github.com/anjali-hole/Project_2024-honors-group3/blob/0b887e8de4aa458faa57711d880b3486ca2ecec6/Code/data_generation.cpp#L7